cell_types_utils.R

```diff
@@ -353,3 +353,11 @@ extract_metadata = function(file){
         idx = idx + 1 
     }
 }
+
+# remove non-aplhanumeric charecters from metadata/marker gene files
+# given a vector of labels, return updated labels
+filter_labels = function(labels, reg_exp){
```

Comment:
Set a sensible default on the regex?

Reply:
@pinin4fjords It's set in a script that invokes this method. We could parameterise it and set a default here.

Reply:
I'd say having non-alphanumerics as the default set to filter out here is a good idea, even if you also supply it explicitly. But if you don't want to do that you should check the value of reg_exp before using it.